### DISCUSSION TITLE
Separate beta and stable releases

### DISCUSSION THREAD

**Discussion by xick** [immich-app/immich#9000] [General]
I believe Immich should consider implementing two separate release channels: a **_beta_** version for those who always desire the latest features and updates, and a **_stable_** version for users who prefer not to constantly update their app and server, either due to time constraints or personal preference.

At present, updates are quite frequent, which is fortunate, but can be challenging to always synchronize the app and server versions. Typically, mobile apps update automatically, while server updates require manual intervention (we've seen several instances where updates required manual adjustments to configs).

**Comment by alextran1502** [comment]
I think this is not possible with us having companion mobile app as well. The goal is to reach stable this year so we are working toward a roadmap for what would it look like for stable release

**Reply by bo0tzz** [reply]
To expand on the reasoning: As indicated by all the warnings, Immich is still very much under heavy development right now. Having separate channels and stability/backwards compatibility take away from that, which we want to avoid for now. Yes there are frequent (and sometimes breaking) updates, but that's the tradeoff for anybody who wants to run Immich right now :)

**Reply by xick** [reply]
Yes of course this approach would require shipping two separate mobile app as well
